quipucords/scanner/network/utils.py

```diff
@@ -89,10 +89,12 @@ def write_inventory(inventory):
     return write_to_yaml(inventory)
 
 
-def create_ansible_objects(inventory_file, forks=50):
+def create_ansible_objects(inventory_file, optional_products_status, forks=50):
```

Comment:
lets just call this `extra_vars` instead of something specific to this user story for use in the future.